### DISCUSSION TITLE
Using an existing ODBC `con` object to connect to remote db

### DISCUSSION THREAD

**Discussion by ghost** [rstudio/pointblank#457] [Ideas]
I'm not certain if this is already feasible, but I couldn't find it from the documentation so would just like to make a suggestion. We've configured our workbench to use an odbc connection to our database, so we just hit "connect" and it gives us a `con` object we can use for querying and such. Would be cool if `db_tbl` could use an object like that rather than have to enter in the db details.

**Comment by rich-iannone** [comment]
Totally agree and I didn’t know about that use case. Would you mind putting this request into a new issue?
